strato

```diff
@@ -474,5 +492,9 @@ while ! [[ -n $(docker ps -q -f name=${hc_container} -f health=healthy) ]];  do
   if [[ $((i % 4)) -eq 0 ]]; then printf '\b\b\b   \b\b\b'; else printf '.'; fi
   sleep 0.3
 done
+if [ "$OAUTH_ENABLED" == "true" ] || [ "$OAUTH_JWT_VALIDATION_ENABLED" == "true" ]
```

Comment:
I think we should keep strato-getting-started backwards compatible with pre-4.5 STRATO version so we only need to set the password for 4.5+.
One simple way for that would be to check if the `/strato/v2.3/password` endpoint exists in vault-wrapper container.